### ISSUE TITLE
DD Tech: Ability to Edit Note or Time Entry

### ISSUE THREAD

**Issue by dr3dgen** [DeskDirector/UserVoice#313]
Can we please have a feature that allows a technician to edit their time entry or note in DD Tech? Sometimes the wrong date or length of time may be inputted, and it would be unfortunate to have to log into CW just to edit a recent entry.

**Comment by Lowell-Syncrety** [comment]
Yes, this is needed. Sometimes information that's put in needs to be removed later out of a ticket, or something was mistyped.

**Comment by iskolares** [comment]
Related request: Ticket #26962

**Comment by drkmccy** [comment]
Yes please. I edit a high percentage of my notes in Connectwise.

**Comment by Peaches33** [comment]
Yes, Please.  There are many times information is updated and it is missed as it many entries down from the original request and thus gets missed.
